bakery/src/tasks/fetch-book-group.js

```diff
@@ -27,15 +28,17 @@ const task = (taskArgs) => {
       inputs: [{ name: bookInput }],
       outputs: [
         { name: contentOutput },
-        { name: resourceOutput }
+        { name: resourceOutput },
+        { name: unusedResourceOutput }
       ],
       params: {
         COLUMNS: 80,
         BOOK_INPUT: bookInput,
         GH_SECRET_CREDS: githubSecretCreds,
         CONTENT_OUTPUT: contentOutput,
         BOOK_SLUGS_URL: bookSlugsUrl,
-        RESOURCE_OUTPUT: resourceOutput
+        RESOURCE_OUTPUT: resourceOutput,
```

Comment:
It looks like this parameter is never consumed and instead we're using `.` in the script invocation.